test/docs/promises.test.js

```diff
@@ -21,6 +23,8 @@ describe('promises docs', function () {
   });
 
   after(async function () {
+    global.Promise = defaultPromise;
```

Comment:
this should work in its current way, though personally i would put this in a `afterEach` instead of just a `after`